flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveTableSourceStatisticsReportTest.java

```diff
@@ -292,20 +294,41 @@ private static void assertHiveTableOrcFormatTableStatsEquals(
         expectedColumnStatsMap.put(
                 "f_string",
                 new ColumnStats.Builder().setMax("def").setMin("abcd").setNullCount(0L).build());
-        expectedColumnStatsMap.put(
-                "f_decimal5",
-                new ColumnStats.Builder()
-                        .setMax(new BigDecimal("223.45"))
-                        .setMin(new BigDecimal("123.45"))
-                        .setNullCount(0L)
-                        .build());
-        expectedColumnStatsMap.put(
-                "f_decimal14",
-                new ColumnStats.Builder()
-                        .setMax(new BigDecimal("123333333355.33"))
-                        .setMin(new BigDecimal("123333333333.33"))
-                        .setNullCount(0L)
-                        .build());
+        if (hiveVersion.startsWith("3.")) {
```

Comment:
You can follow the check logic in `HiveModuleTest#verifyNumBuiltInFunctions`.

Reply:
> You can follow the check logic in `HiveModuleTest#verifyNumBuiltInFunctions`.

Done!